CryptoPrices/Sources/MyCoin/Sources/MyCoin/CurrentPriceSectionView/CurrentPriceSection.swift

```diff
@@ -0,0 +1,58 @@
+//
+//  CurrentPriceSection.swift
+//  MyCoin
+//
+//  Created by Minh Pham on 22/12/2022.
+//
+
+import Styleguide
+import SwiftUI
+
+struct CurrentPriceSection: View {
+
+    var body: some View {
+        VStack(spacing: 8.0) {
+            // TODO: - Remove dummy
+            Images.icEth.swiftUIImage
+                .resizable()
+                .aspectRatio(contentMode: .fit)
+                .frame(width: 60.0, height: 60.0)
+                .clipShape(Circle())
+
+            VStack(spacing: 4.0) {
+                // TODO: Remove hard-coded data
+                Text(7_273_291, format: .dollarCurrency)
```

Comment:
It's optional. If we have a decimal number (for exp: `7_273.291`) like `2.41` below, it would be nice :D.

Reply:
I think it would not be a big deal, as we will need to update to real data anyway. Let's keep it as-is 🙏